commit a35e8d892628d01e50af23aee5ba501be426baf6
Author: Tony Hutter <hutter2@llnl.gov>
Date:   Wed Jun 17 09:45:38 2026 -0700

    Linux 7.1 compat: META (#18682)
    
    Update the META file to reflect compatibility with the 7.1
    kernel.
    
    Signed-off-by: Tony Hutter <hutter2@llnl.gov>
    Signed-off-by: Rob Norris <rob.norris@truenas.com>
    Reviewed-by: Chris Longros <chris.longros@gmail.com>

diff --git a/META b/META
index ab1b8955d..3c688e54a 100644
--- a/META
+++ b/META
@@ -6,5 +6,5 @@ Release:       1
 Release-Tags:  relext
 License:       CDDL
 Author:        OpenZFS
-Linux-Maximum: 7.0
+Linux-Maximum: 7.1
 Linux-Minimum: 4.18
